src/core/layout/qgslayoutexporter.h

```diff
@@ -336,7 +336,7 @@ class CORE_EXPORT QgsLayoutExporter
        *
        * \since QGIS 3.14
        */
-      bool exportLayersAsSeperateFiles = false;
+      bool exportLayersAsSeparateFiles = false;
```

Comment:
@nyalldawson this is basically API change.. do we bother?

Reply:
Bah -- better add a "TODO QGIS 4" note and leave it at that...